oak/server/rust/oak_runtime/src/node/grpc_server.rs

```diff
@@ -23,23 +33,18 @@ use std::{
     thread::{self, JoinHandle},
 };
 
-use oak_abi::{
-    grpc::encap_request, label::Label, proto::oak::encap::GrpcRequest, ChannelReadStatus, OakStatus,
-};
-
-use crate::{metrics::METRICS, pretty_name_for_thread, runtime::RuntimeProxy, Handle};
-
 /// Struct that represents a gRPC server pseudo-Node.
 ///
 /// For each gRPC request from a client, gRPC server pseudo-Node creates a pair of temporary
 /// channels (to write a request to and to read a response from) and passes corresponding handles to
 /// the [`GrpcServerNode::channel_writer`].
+#[derive(Clone)]
```

Comment:
Why do we need `Clone` ?

Reply:
That's a question for you I guess :) I just removed the manual implementation and replaced it with `derive`.